packages/manager/apps/vrack-services/src/api/vrack/hook.ts

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable import/prefer-default-export */
+import { useQueries, useQuery } from '@tanstack/react-query';
+import {
+  getVrackList,
+  getVrackListQueryKey,
+  getVrackServiceAllowedServices,
+  getVrackServiceAllowedServicesQueryKey,
+} from './get';
+
+export const useVrackServicesAllowedVrack = (vrackServicesId?: string) => {
```

Comment:
Naming : useAllowedVrackServices

Reply:
It's not a good name since the hook returns the list of allowed vRack for a vRack Services

```
useVrackServicesAllowedVrackList
```
or
```
useAllowedVrackList
```

Reply:
Yes second one is ok for me useAllowedVrackList

Reply:
https://github.com/ovh/manager/pull/10418